dns/modules/ssl_cert/README.md

```diff
@@ -0,0 +1,13 @@
+# ssl_cert
```

Comment:
There is already a certificate module in this repo I can see ... I wonder whether we can just adapt that one, rather than bringing in this one ... i'll have a go and push that in another draft PR to have a look at.

Reply:
Like this: https://github.com/wellcomecollection/platform-infrastructure/pull/111